core/utils.py

```diff
@@ -235,6 +235,24 @@ def dict_from_yaml(yaml_str: str) -> Dict[str, Any]:
         raise InvalidInputException(e)
 
 
```

Comment:
@DiptoChakrabarty change pyyaml version in this file to 6.0

Reply:
okay modified